data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+			`	Once she's done, she says you're free to roam the camp as you wait for the "battle" to begin. A few minutes pass before a loud whistle blows, and the Kimek in charge of the camp tells everyone that it's begun. She leads the charge up the hill, with most of those from the camp following her. Others split into smaller groups and get closer to the trees that separate the sets, trying to approach the fort from the sides.`
+			choice
+				`	(Charge with the main group.)`
+					goto attw
+				`	(Take the longer side route.)`
+					goto attr
+			label attw
+			`	You follow the lead of the Kimek and start a run for it up the hill, directly at the fort. Once you get closer, however, you see the defenders pouring down what looks like water. As it flows down the hill to meet your steps, you lose your balance and trip, sliding a bit down the hill. You get up, but, try it as you may, you cannot find footing with the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek aren't so lucky and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`
+			choice
+				`	(Try and find another way in.)`
+					goto attwr
+				`	(Help the others bust the gate open.)`
+			`	You start pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate. They take turns with others when they get tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate. It takes some good damage with each time, gradually budging more. The battering ram cannot hit it enough times, however, as the whistle from the start of the event sounds again. It evidently means that time has run out and that the defenders have won.`
```

Comment:
```suggestion
			`	You start pushing along with the others, the Saryds joining you from below, and the Kimek and Arachi crawling up to push at the top of the gate. They take turns with each others when one gets tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate finally begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it against the gate. The gate shudders with each hit and gradually begins to give even more, but even the battering ram team cannot break through fast enough; as the whistle from the start of the event sounds again. Evidently this means that time has run out, and the defenders have won.`
```